weblab/templates/entities/entity_confirm_delete.html

```diff
@@ -8,6 +8,7 @@
     <p>Are you sure you want to delete all versions of {{ object.display_type }} "{{ object }}"?</p>
     <p><strong>This operation cannot be undone.</strong></p>
     <button>Confirm</button>
+    <a class="button" href="{% entity_url 'detail' object %}" title="Do not delete">Cancel</a>
```

Comment:
@jonc125 I tried this as per the note on the issue but it errors with 'Entity' object has no attribute 'url_type'

Reply:
Hah, good point! I hadn't anticipated that because we use `display_type` above, but actually that gives an empty string because it's not set for the generic `Entity` base class either. So maybe `href="javascript:history.back()"` would be better? Even if it does cheat a little!

Reply:
And we should probably get rid of the `display_type` reference too.